content/post/frontend_talks_2017.md

```diff
@@ -0,0 +1,142 @@
+---
+author: erizocosmico
+date: 2017-07-10
+title: "source{d} tech talks, frontend series"
+image: /post/frontend-talks/header-pic.jpg
+description: "Once every few months, source{d} organizes small conferences around a very specific topic. On June, 24th the topic was frontend and the talks were hosted in our Madrid offices. Almost 50 people joined us for a day full of things to learn about frontend technologies."
```

Comment:
Yeah, I also thought that sounded weird.

Something like `a full day to learn something new about frontend technologies`?